common/static/js/src/logger.js

```diff
@@ -0,0 +1,91 @@
+;(function() {
+    'use strict';
+    var Logger = (function() {
+        // listeners[event_type][element] -> list of callbacks
+        var listeners = {},
+            sendRequest, has;
+
+        sendRequest = function(data, options) {
+            var request = $.ajaxWithPrefix ? $.ajaxWithPrefix : $.ajax;
```

Comment:
No, I don't know what the `EDX_ROOT_URL` setting is intended to be used for. Seems like it's only used for `<meta name="path_prefix">` in the HTML for our pages -- and I don't know what that is for. SEO, perhaps?